### PR TITLE
Update new output for pyCheck program for ELK monitoring

### PR DESCRIPTION
**Pull request by hongson1981**
# Output format like this :
# -> Dec 23 12:11:43 78.159.113.57 ping 8.8.8.8: PASS 13.23
# Dec 23 12:11:43 : timestamp
# 78.159.113.57: host
# ping 8.8.8.8: checkpoint
# PASS: result
# 13.23: round trip time in ms

### CODE REVIEW

File: HealthCheck/pyCheck.py [hunk 1]
```diff
@@ -15,50 +13,64 @@
...
+        for byte_block in iter(lambda: f.read(4096), b""):
             hashfunc.update(byte_block)
     return hashfunc.hexdigest()
 
+
 def main(args):
     retcode = 0
+# Get public ip address of host
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))
```
Comment (MariuszFerdyn) [inline HealthCheck/pyCheck.py:29]: 8.8.8.8, 80 - hardcoded?

**Review by MariuszFerdyn — CHANGES_REQUESTED** [review]

**Comment by hongson1981** [comment]
@MariuszFerdyn , @mahnouman 
UPdate python code to resolve hardcode issue

**Comment by MariuszFerdyn** [comment]
ok Can be merge.

**Comment by MariuszFerdyn** [comment]
Can be merge.

**Pull request merged**